starter-code/iteration1/src/app/app.component.ts

```diff
@@ -6,5 +6,26 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'app works!';
+  title = 'My Favorite Cities!';
+
+  my1stPhoto: string = 'http://www.visitflorida.com/content/dam/visitflorida/en-us/images/cities/miami/Downtown2011_08.jpg';
+  my1stPhotoComment: string = 'Miami';
+  my2ndPhoto: string = 'http://www.seetorontonow.com/wp-content/uploads/2017/10/toronto-fall-sunset.jpg';
+  my2ndPhotoComment: string = 'Toronto';
+  my3rdPhoto: string = 'https://cdn.flixbus.de/city_description_images/vienna-nightlife.jpg';
+  my3rdPhotoComment: string = 'Vienna';
+  // photos: Array<Object> = [{
+  //     id: 1,
+  //     Image: string = 'http://placekitten.com/g/300/300',
+  //     comment: 'First Photo'
+  // },{
+  //     id: 2,
+  //     Image: string = 'http://placekitten.com/g/300/300',
+  //     comment: 'Second Photo'
+  // },{
+  //     id: 3,
+  //     Image: string = 'http://placekitten.com/g/300/300',
+  //     comment: 'Third Photo'
+  // }];
+
 }
```

Comment:
The commented out part is the right way to do it. We want our code to be as organized as possible, as short as possible and as readable as possible. You know how to apply string interpolation, that's good. Please refer to lessons and the way we did it at the class and ask me any questions you might have along the way.